client/src/curriculum/scrim-inline.js

```diff
@@ -127,9 +139,13 @@ class ScrimInline extends LitElement {
     `;
   }
 
-  #toggle(e: MouseEvent) {
+  /**
+   * @param {MouseEvent} e
+   */
+  #toggle(e) {
     if (e.target) {
-      (e.target as HTMLElement).dataset.glean =
+      /** @type {HTMLElement} */
+      (e.target).dataset.glean =
```

Comment:
TIL: This declares the type of `(e.target)`?

Reply:
Yeah, though it's very weird syntax isn't it? I've changed it to a `instanceof HTMLElement` check for better readability and safety